Dockerfile

```diff
@@ -44,10 +44,9 @@ RUN apt-get install -y gdb gdbserver
 # These packages allow for a better integration for some containers, for example
 # daemontools provides envdir, which is very convenient for passing backup
 # environment variables around.
-RUN apt-get update && apt-get install -y dumb-init daemontools
+RUN apt-get install -y dumb-init daemontools
```

Comment:
Shouldn't we run `apt-get update` at least once to get new packages? I see it's remove in two places and I am not sure if we call it from some other place

Reply:
We call it on line 27